sdks/python/apache_beam/typehints/trivial_inference_test.py

```diff
@@ -299,7 +299,7 @@ def m(self, x):
         return x
 
     self.assertReturnType(int, lambda: A().m(3))
-    self.assertReturnType(float, lambda: A.m(A(), 3.0))
+    #self.assertReturnType(float, lambda: A.m(A(), 3.0))
```

Comment:
why did we comment this test?

Reply:
I had commented that out to diagnose an issue with the first test case, just forgot to reinstate it. Fixed